chsdi/locale/it/LC_MESSAGES/chsdi.po

```diff
@@ -4940,6 +4940,9 @@ msgstr "Centrali nucleari"
 msgid "ch.bfe.kleinwasserkraftpotentiale"
 msgstr "Potenz. piccole centr. idraul."
 
+msgid "ch.bfe.ladestellen-elektromobilitaet"
+msgstr "Ladeinfrastruktur Elektromobilität"
```

Comment:
No translation?

Reply:
ch.bfe.ladestellen-elektromobilitaet is the DIEMO layer and only on dev. No problem with that